src/native/corehost/fxr/fx_resolver.messages.cpp

```diff
@@ -135,6 +124,30 @@ void fx_resolver_t::display_missing_framework_error(
         trace::error(_X("No frameworks were found."));
     }
 
+    std::vector<std::pair<pal::architecture, std::vector<framework_info>>> other_arch_framework_infos;
+    install_info::enumerate_other_architectures(
```

Comment:
This would be a bit clearer if `enumerate_other_architectures` could return the `std::vector`. I'm assuming we could get rid of the byref capture (`[&]`) if that is the case. If not, then this is fine.